docs/book/starter-guide/collaborate/roles-and-permissions.md

```diff
@@ -0,0 +1,75 @@
+---
+description: How roles and permissions work within ZenML
+---
+{% hint style="warning" %}
+Permissions and roles are an alpha feature of ZenML and are in active 
+development.
+{% endhint %}
+
+# Permissions
+
+The following global permissions are available within ZenML.
+
+**read**: Read permission on all resources.
+**write**: Write permission on all resources.
+**me**: Permission for a user to edit their own user metadata
+(username, password, etc.)
+
+To view the available permissions through the CLI simply run:
+
+`zenml permission list`
+
+# Roles
+
+Users are granted **Permissions** through **Roles**. You can list all roles
+with their associated permissions by running.
+
+`zenml role list`
+
+## Default Roles
+
+By default, ZenML comes with two roles.
+
+**admin**: The admin role grants all permissions globally.
+
+**guest**: The guest role lets users change their own user data and grants 
+general global read access.
+
+## Custom Roles
+
+You can also create your own roles through the CLI:
+
+`zenml role create <ROLE_NAME> -p read -p write -p me`
```

Comment:
Usage: zenml role create [OPTIONS] ROLE_NAME

  Create a new role.

Options:
  -w, --write
  -r, --read
  -m, --me

Reply:
good point, I'll add it here, although users can also run `zenml role create --help` this will generate this:

```
Usage: zenml role create [OPTIONS] ROLE_NAME

  Create a new role.

Options:
  -p, --permissions [write|read|me]
                                  Name of permission to attach to this role.
  --help                          Show this message and exit.
```

Reply:
Just as a headsup @Tellz777 all of this will only be relevant zenml>=0.22.0

Reply:
@AlexejPenner Basically, for me this is the output of `zenml role create --help`. Im using the latest release version (0.22.0)